Libs/MRML/Core/vtkMRMLVolumeArchetypeStorageNode.cxx

```diff
@@ -281,6 +282,36 @@ void ApplyImageSeriesReaderWorkaround(vtkMRMLVolumeArchetypeStorageNode * storag
     }
   }
 }
+
+//----------------------------------------------------------------------------
+bool IsVolumeCoordinateSystemLeftHanded( const vtkSmartPointer<vtkMatrix4x4> &ijkToRASMatrix)
+{
+  // Check if the determinant of the direction matrix is less than 0.
+  vtkNew<vtkMatrix3x3> direction;
+  for(int i=0; i<3; i++){
+    for (int j=0; j<3; j++){
+      direction->SetElement(i, j, ijkToRASMatrix->GetElement(i, j));
+    }
+  }
```

Comment:
alternatively you could have used `vtkMath::Determinant(itkToRASMatrix->GetElement(0,0), ...)`

Reply:
Yes, but I'm personally not a fan of having 9 `itkToRASMatrix->GetElement` on a few lines. It fills less readable than two for loops.

Reply:
Please use vtkAddon for this - https://github.com/Slicer/vtkAddon/blob/960c2e3c8aa5155951930b53f1590f3db8f94d80/vtkAddonMathUtilities.h#L65

Reply:
Thx for the info. Didn't know this utility function.
I just updated the code.